cmd/stack_export.go

```diff
@@ -66,6 +67,8 @@ func newStackExportCmd() *cobra.Command {
 			return nil
 		}),
 	}
+	cmd.PersistentFlags().StringVarP(
+		&stackName, "stack", "s", "", "The name of the stack to operate on. Defaults to the current stack")
```

Comment:
Does it make sense to update the other commands that accept `--stack` to also use this consistent description while we are at it?

Reply:
Done